ai-ml/jark-stack/terraform/eks.tf

```diff
@@ -147,5 +147,66 @@ module "eks" {
         Name = "gpu-node-grp"
       })
     }
+
+    # # This nodegroup can be used for P4/P5 instances with, or without, a Capacity Reservation.
+    # #
```

Comment:
@alanty is this comment required? We need to make sure that we have the instructions when to remove the comments of this section, might want to add into the website folder documentation for the JARK stack

Reply:
@lusoal, the JARK stack currently missing documentation for the DoEKS website. I agree that adding a section to explain how to enable and deploy the P5 nodegroup with EFA and CBR would be beneficial. We can do that as a part JARK deployment Website doc.